addons/inventory/XEH_preStart.sqf

```diff
@@ -16,6 +16,10 @@ uiNamespace setVariable [QGVAR(ItemKeyCache), _allItems apply {
     if (_picture select [0,1] == "\") then {
         _picture = _picture select [1];
     };
+    if (count _picture > 0 && !(_picture regexMatch ".*?\.paa")) then { // handle missing file extension
+        if (!fileExists (_picture + ".paa")) exitWith {};
+        _picture = _picture + ".paa";
+    };
```

Comment:
Do we want to check for all items if they have a valid `.paa` file and give invalid ones a default picture?